utils.py

```diff
@@ -224,6 +226,17 @@ async def _trigger_create_thread(bot, member, message, open_type, is_mention=Fal
             await member.send('You are not banned from /r/NintendoSwitch and have been kicked from the ban appeal server.')
             await appealGuild.fetch_member(member.id).kick(reason='Member is not banned on /r/NintendoSwitch')
 
+        else:
+            pun = db.find_one({'type': 'appealdeny', 'active': True})
```

Comment:
Maybe this whole logic for checking if someone is not yet ready to appeal and/or not in the server should be a function instead of duplicated in  _trigger_create_thread() in utils.py and  on_member_join() in bot.py.